docs/developers/connectors/architecture.md

```diff
@@ -146,3 +146,70 @@ For more details on how to begin implementing the components, please refer to th
 ## Protocol Connector Components Overview\ [TBD\]
 
 Coming soon.
+
+## Fee Accounting
+The `BudgetChecker` uses the information from a `TradeFeeSchema` to generate a specific instance of `TradeFeeBase` that is then applied to an `OrderCandidate` in order to asses the order's effects on account balances.
+
+### TradeFee
+
+The `TradeFee` object contains the necessary information to account for fees when estimating an order's impact on account balances.
+
+#### TradeFeeSchema
+
+Contains the necessary information to build the `TradeFee` object.
+For both makers and takers specifies percent and fixed fees, and tokens in which the fees are paid.
+Exchanges must specify their respective default schemas inside their `[exchange]_utils.py` files:
+```python
+DEFAULT_FEES = TradeFeeSchema(
+    maker_percent_fee_decimal=Decimal("0.001"),
+    taker_percent_fee_decimal=Decimal("0.001"),
+)
```

Comment:
The code snipped is not closed here, but it won't let me insert the closing ticks in a suggestion

Reply:
Fixed